packages/backend-lib/src/types.ts

```diff
@@ -434,15 +434,9 @@ export const ResendEvent = Type.Object({
     from: Type.String(),
     subject: Type.String(),
     to: Type.Array(Type.String()),
+    tags: Type.Record(Type.String(), Type.String()),
```

Comment:
Can you make a

```typescript
Type.Object({
  workspaceId: Type.Optional(Type.String()),
  runId: Type.Optional(Type.String()),
  messageId: Type.Optional(Type.String()),
  userId: Type.Optional(Type.String()),
  templateId: Type.Optional(Type.String()),
  journeyId: Type.Optional(Type.String()),
  anonymousId: Type.Optional(Type.String()),
})
```

Instead? That way if for some reason they start sending non-string values (that we're not tracking) we won't error. Has happened to me from other email webhooks.